qamomile/core/converters/qaoa.py

```diff
@@ -30,13 +30,18 @@
 """
 
 import typing as typ
+import math
 import jijmodeling_transpiler.core as jmt
 import qamomile.core.bitssample as qm_bs
 import qamomile.core.circuit as qm_c
 import qamomile.core.operator as qm_o
 from qamomile.core.converters.converter import QuantumConverter
 
 
+EPSILON = 1e-15
+def is_close_zero(value: float) -> bool:
```

Comment:
@Jacomichi 
The dependency where the `is_close_zero function`, which is used not only in qaoa but also in other converters, is written in qaoa and called from qrao and others, may cause unintended bugs related to import order. Therefore, please prepare `converters/utils.py` or similar and write the function there to be imported.

Reply:
I have created `converter/utils.py` and moved `is_close_zero`